openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -108,6 +111,8 @@ def __init__(self, *args, **kwargs):
         self._project_settings = get_project_settings(
             legacy_io.Session["AVALON_PROJECT"])
 
+
+
         # project_settings/maya/create/CreateRender/aov_separator
```

Comment:
Isn't this too many empty lines? Why isn't the hound reacting?
```suggestion
            legacy_io.Session["AVALON_PROJECT"])

        # project_settings/maya/create/CreateRender/aov_separator
```

Reply:
Fixed!